strings/valentines.json

```diff
@@ -73,11 +75,27 @@
 		"Are you wearing space pants? I wish you weren't.",
 		"You give me a positive moodlet.",
 		"Let's make like swarmers and reproduce.",
-		"If you were a drone, would you be free tonight?"
+		"If you were a drone, would you be free tonight?",
+		"Get the paramedics -- your body put me into crit!",
+		"Come to the incinerator and be my maint squeeze.",
+		"Come to the courtroom so we can spend some time alone.",
+		"I'm the gray tide -- may I take your hand tele?",
+		"If I were a clown car, would you come inside me?",
+		"If you were telecoms, would you go down on me?",
+		"Don't touch any potassium, I might get you wet.",
```

Comment:
Bit much dont you think? The rest are are mostly corny pick up lines

Reply:
18+

Reply:
I meant all three of these